src/Components/RecsListTable/index.js

```diff
@@ -296,6 +309,18 @@ const RecsListTable = () => {
     },
   ];
 
+  const onSort = (_e, index, direction) => {
+    const sortRecsDirection =
+      direction === SortByDirection.asc
+        ? buildFilteredRows(recs, filters, index)
+        : buildFilteredRows(recs, filters, index).reverse();
+    setSortBy({
```

Comment:
Just to add: once you move the sorting params to the redux store, you should call dispatch instead of the set state function.